_posts/2021-05-09-april-rubygems-update.md

```diff
@@ -0,0 +1,43 @@
+---
+title: April 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in April.
+
+## RubyGems News
+This Month in RubyGems, we released a new version for **RubyGems(3.2.16)** and **Bundler(2.2.16)**. 
```

Comment:
We also released Rubygems 3.2.17 and Bundler 2.2.17, the 5th of May. Releases didn't fall inside April, but most of the work for them was done during April, so I guess it's worth mentioning them.

Reply:
Yes for sure, they are indeed worth mentioning